examples/networking/subnet/main-customdns.tf

```diff
@@ -0,0 +1,6 @@
+resource "vkcs_networking_subnet" "custom_subnet_1" {
+  name       = "custom_subnet_1"
+  network_id = vkcs_networking_network.app.id
+  dns_nameservers = ["8.8.8.8"]
```

Comment:
Add a comment to explain that here we define DNS'es instead of built in ones. And mention that resources in this subnet will not be available via thier private DNS names.

Reply:
added